contracts/pm/mixins/MixinReserve.sol

```diff
@@ -20,7 +20,7 @@ contract MixinReserve is MContractRegistry, MReserve {
     mapping (address => Reserve) internal reserves;
 
     /**
-     * @dev Returns info about a reserve
+     * @notice Returns info about a reserve
```

Comment:
All of the functions in this file are internal and are never used by an external account so it feels like `@dev` should be used instead.

Reply:
fixed in b79bb67